puppet/modules/freight/templates/vhost.erb

```diff
@@ -1,3 +0,0 @@
-<Directory />
```

Comment:
@GregSutcliffe was the some reason to set this to `Directory /` instead of `Location /`?